js/constants/appConfig.js

```diff
@@ -149,6 +149,7 @@ module.exports = {
     'bookmarks.toolbar.showOnlyFavicon': false,
     'payments.enabled': false,
     'payments.notifications': false,
+    'payments.non-verified-publishers': true,
```

Comment:
Could you update this to be more descriptive? Maybe `payments.allow-non-verified-publishers`?